src/bucket.ts

```diff
@@ -522,6 +522,13 @@ class Bucket extends ServiceObject {
     // Allow for "gs://"-style input, and strip any trailing slashes.
     name = name.replace(/^gs:\/\//, '').replace(/\/+$/, '');
 
+    const requestQueryObject: {userProject?: string} = {};
```

Comment:
Should we just declare an external interface for this? Not sure if it will ever be useful for the user or not.

Reply:
While `serviceObject(config.methods[method].reqOpts` follows the request library's signature, I would still consider it private behavior. I wouldn't document it, since we might come up with something different in the future, and I'm not sure the user would understand exactly what it does / what side effects it has.